app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(line: str) -> None:
+    com, file1, file2 = line.split()
```

Comment:
use full names for variables - command not com
also it is better to use more informative variables not file1/file2 - new_file, old_file